{"expand":"renderedFields,names,schema,operations,editmeta,changelog,versionedRepresentations","id":"16630","self":"https://eucalyptus.atlassian.net/rest/api/2/issue/16630","key":"EUCA-3908","fields":{"issuetype":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issuetype/1","id":"1","description":"A problem which impairs or prevents the functions of the product.","iconUrl":"https://eucalyptus.atlassian.net/secure/viewavatar?size=xsmall&avatarId=14803&avatarType=issuetype","name":"Bug","subtask":false,"avatarId":14803},"timespent":null,"customfield_13100":null,"project":{"self":"https://eucalyptus.atlassian.net/rest/api/2/project/10000","id":"10000","key":"EUCA","name":"Eucalyptus","avatarUrls":{"48x48":"https://eucalyptus.atlassian.net/secure/projectavatar?pid=10000&avatarId=10011","24x24":"https://eucalyptus.atlassian.net/secure/projectavatar?size=small&pid=10000&avatarId=10011","16x16":"https://eucalyptus.atlassian.net/secure/projectavatar?size=xsmall&pid=10000&avatarId=10011","32x32":"https://eucalyptus.atlassian.net/secure/projectavatar?size=medium&pid=10000&avatarId=10011"}},"customfield_11000":null,"customfield_11001":null,"fixVersions":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/version/10112","id":"10112","description":"Eucalyptus 3.2.0","name":"3.2.0","archived":false,"released":true,"releaseDate":"2012-12-19"}],"aggregatetimespent":null,"customfield_10310":null,"resolution":{"self":"https://eucalyptus.atlassian.net/rest/api/2/resolution/1","id":"1","description":"A fix for this issue is checked into the tree and tested.","name":"Fixed"},"customfield_10311":null,"customfield_11401":null,"customfield_11400":null,"customfield_10302":null,"customfield_10104":["com.atlassian.greenhopper.service.sprint.Sprint@10543809[id=9,rapidViewId=<null>,state=CLOSED,name=3.2 Sprint 6,goal=<null>,startDate=2012-10-23T16:29:16.193Z,endDate=2012-11-08T17:29:16.193Z,completeDate=2012-11-10T22:13:51.878Z,sequence=9]","com.atlassian.greenhopper.service.sprint.Sprint@1e13d37c[id=10,rapidViewId=<null>,state=CLOSED,name=3.2 Sprint 7,goal=<null>,startDate=2012-11-10T22:18:20.990Z,endDate=2012-12-04T19:02:20.990Z,completeDate=2012-12-26T19:14:23.804Z,sequence=10]"],"customfield_10500":{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10403","value":"Not Applicable","id":"10403"},"customfield_10105":{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10101","value":"No","id":"10101"},"customfield_10303":null,"customfield_10700":null,"customfield_10304":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/version/10112","id":"10112","description":"Eucalyptus 3.2.0","name":"3.2.0","archived":false,"released":true,"releaseDate":"2012-12-19"}],"customfield_10701":null,"customfield_10306":null,"customfield_10702":null,"customfield_10703":null,"resolutiondate":"2012-12-04T23:59:18.549-0600","customfield_10704":null,"customfield_10308":null,"customfield_10309":null,"customfield_10705":null,"workratio":-1,"lastViewed":null,"watches":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/EUCA-3908/watchers","watchCount":3,"isWatching":false},"created":"2012-10-22T19:16:43.364-0500","customfield_12000":null,"priority":{"self":"https://eucalyptus.atlassian.net/rest/api/2/priority/3","iconUrl":"https://eucalyptus.atlassian.net/images/icons/priorities/major.svg","name":"Major","id":"3"},"customfield_12400":null,"customfield_12201":null,"customfield_10300":null,"customfield_12600":"{}","customfield_10102":null,"customfield_10301":null,"labels":["QA-assigned"],"customfield_11700":null,"aggregatetimeoriginalestimate":null,"timeestimate":null,"versions":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/version/10706","id":"10706","name":"3.2.0-devel","archived":false,"released":true}],"customfield_11901":null,"issuelinks":[],"assignee":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=vasya","name":"vasya","key":"vasya","accountId":"557058:1c63481a-0634-4ae7-9004-cb0f143e141f","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue"},"displayName":"Vasiliy Kochergin","active":false,"timeZone":"America/Los_Angeles"},"updated":"2013-01-09T19:57:12.337-0600","status":{"self":"https://eucalyptus.atlassian.net/rest/api/2/status/6","description":"The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.","iconUrl":"https://eucalyptus.atlassian.net/images/icons/statuses/generic.png","name":"Closed","id":"6","statusCategory":{"self":"https://eucalyptus.atlassian.net/rest/api/2/statuscategory/3","id":3,"key":"done","colorName":"green","name":"Done"}},"components":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/component/10013","id":"10013","name":"Reporting"}],"timeoriginalestimate":null,"customfield_13000":null,"description":"If an instance is recreated after the original instance create event has been exported and deleted then an error will occur when next exporting and importing to the data warehouse:\r\n\r\n{code}\r\n# eucadw-import-data -e /tmp/export.xml -E\r\nDatabase password: \r\n2012-10-22 16:35:57 ERROR | Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-10-22 15:28:04.736000 -07:00:00', '1350944884734', 'PARTI00', 'i-627A4396', 'm1.small', 'XT7RLQVPQXT5LAACAXM3G', 'd8e86942-47ab-4c2-9b24-9280f29dd1f8', '4128b3273a8a884e013a8a9708000124') was aborted.  Call getNextException to see the cause.\r\n0 [main] ERROR org.hibernate.util.JDBCExceptionReporter  - Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-10-22 15:28:04.736000 -07:00:00', '1350944884734', 'PARTI00', 'i-627A4396', 'm1.small', 'XT7RLQVPQXT5LAACAXM3G', 'd8e86942-47ab-4c2-9b24-9280f29dd1f8', '4128b3273a8a884e013a8a9708000124') was aborted.  Call getNextException to see the cause.\r\n2012-10-22 16:35:57 ERROR | ERROR: duplicate key value violates unique constraint \"reporting_instance_create_events_uuid_key\"\r\n                          |   Detail: Key (uuid)=(d8e86942-47ab-4c2-9b24-9280f29dd1f8) already exists.\r\n2 [main] ERROR org.hibernate.util.JDBCExceptionReporter  - ERROR: duplicate key value violates unique constraint \"reporting_instance_create_events_uuid_key\"\r\n  Detail: Key (uuid)=(d8e86942-47ab-4c2-9b24-9280f29dd1f8) already exists.\r\n2012-10-22 16:35:57 ERROR | Could not synchronize database state with session\r\norg.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:262)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:178)\r\n\tat org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)\r\n\tat org.hibernate.event.def.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:51)\r\n\tat org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1206)\r\n\tat org.hibernate.impl.SessionImpl.managedFlush(SessionImpl.java:375)\r\n\tat org.hibernate.transaction.JDBCTransaction.commit(JDBCTransaction.java:137)\r\n\tat org.hibernate.ejb.TransactionImpl.commit(TransactionImpl.java:76)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx$TxState.commit(Entities.java:846)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx.commit(Entities.java:704)\r\n\tat com.eucalyptus.reporting.export.Import.transactional(Import.java:166)\r\n\tat com.eucalyptus.reporting.export.Import.doImport(Import.java:101)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:78)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:70)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.runCommand(ImportCommand.java:45)\r\n\tat com.eucalyptus.reporting.dw.commands.CommandSupport.run(CommandSupport.java:77)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.main(ImportCommand.java:67)\r\nCaused by: java.sql.BatchUpdateException: Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-10-22 15:28:04.736000 -07:00:00', '1350944884734', 'PARTI00', 'i-627A4396', 'm1.small', 'XT7RLQVPQXT5LAACAXM3G', 'd8e86942-47ab-4c2-9b24-9280f29dd1f8', '4128b3273a8a884e013a8a9708000124') was aborted.  Call getNextException to see the cause.\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement$BatchResultHandler.handleError(AbstractJdbc2Statement.java:2621)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1837)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:407)\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement.executeBatch(AbstractJdbc2Statement.java:2754)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:616)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.invoke(ProxyStatement.java:100)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.intercept(ProxyStatement.java:57)\r\n\tat org.postgresql.core.BaseStatement$$EnhancerByProxool$$4c42528e.executeBatch(<generated>)\r\n\tat org.hibernate.jdbc.BatchingBatcher.doExecuteBatch(BatchingBatcher.java:70)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:268)\r\n\t... 17 more\r\n3 [main] ERROR org.hibernate.event.def.AbstractFlushingEventListener  - Could not synchronize database state with session\r\norg.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:262)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:178)\r\n\tat org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)\r\n\tat org.hibernate.event.def.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:51)\r\n\tat org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1206)\r\n\tat org.hibernate.impl.SessionImpl.managedFlush(SessionImpl.java:375)\r\n\tat org.hibernate.transaction.JDBCTransaction.commit(JDBCTransaction.java:137)\r\n\tat org.hibernate.ejb.TransactionImpl.commit(TransactionImpl.java:76)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx$TxState.commit(Entities.java:846)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx.commit(Entities.java:704)\r\n\tat com.eucalyptus.reporting.export.Import.transactional(Import.java:166)\r\n\tat com.eucalyptus.reporting.export.Import.doImport(Import.java:101)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:78)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:70)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.runCommand(ImportCommand.java:45)\r\n\tat com.eucalyptus.reporting.dw.commands.CommandSupport.run(CommandSupport.java:77)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.main(ImportCommand.java:67)\r\nCaused by: java.sql.BatchUpdateException: Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-10-22 15:28:04.736000 -07:00:00', '1350944884734', 'PARTI00', 'i-627A4396', 'm1.small', 'XT7RLQVPQXT5LAACAXM3G', 'd8e86942-47ab-4c2-9b24-9280f29dd1f8', '4128b3273a8a884e013a8a9708000124') was aborted.  Call getNextException to see the cause.\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement$BatchResultHandler.handleError(AbstractJdbc2Statement.java:2621)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1837)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:407)\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement.executeBatch(AbstractJdbc2Statement.java:2754)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:616)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.invoke(ProxyStatement.java:100)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.intercept(ProxyStatement.java:57)\r\n\tat org.postgresql.core.BaseStatement$$EnhancerByProxool$$4c42528e.executeBatch(<generated>)\r\n\tat org.hibernate.jdbc.BatchingBatcher.doExecuteBatch(BatchingBatcher.java:70)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:268)\r\n\t... 17 more\r\nError processing command: Could not execute JDBC batch update\r\n2012-10-22 16:35:57 ERROR | Error processing command\r\norg.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:262)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:178)\r\n\tat org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)\r\n\tat org.hibernate.event.def.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:51)\r\n\tat org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1206)\r\n\tat org.hibernate.impl.SessionImpl.managedFlush(SessionImpl.java:375)\r\n\tat org.hibernate.transaction.JDBCTransaction.commit(JDBCTransaction.java:137)\r\n\tat org.hibernate.ejb.TransactionImpl.commit(TransactionImpl.java:76)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx$TxState.commit(Entities.java:846)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx.commit(Entities.java:704)\r\n\tat com.eucalyptus.reporting.export.Import.transactional(Import.java:166)\r\n\tat com.eucalyptus.reporting.export.Import.doImport(Import.java:101)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:78)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:70)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.runCommand(ImportCommand.java:45)\r\n\tat com.eucalyptus.reporting.dw.commands.CommandSupport.run(CommandSupport.java:77)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.main(ImportCommand.java:67)\r\nCaused by: java.sql.BatchUpdateException: Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-10-22 15:28:04.736000 -07:00:00', '1350944884734', 'PARTI00', 'i-627A4396', 'm1.small', 'XT7RLQVPQXT5LAACAXM3G', 'd8e86942-47ab-4c2-9b24-9280f29dd1f8', '4128b3273a8a884e013a8a9708000124') was aborted.  Call getNextException to see the cause.\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement$BatchResultHandler.handleError(AbstractJdbc2Statement.java:2621)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1837)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:407)\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement.executeBatch(AbstractJdbc2Statement.java:2754)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:616)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.invoke(ProxyStatement.java:100)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.intercept(ProxyStatement.java:57)\r\n\tat org.postgresql.core.BaseStatement$$EnhancerByProxool$$4c42528e.executeBatch(<generated>)\r\n\tat org.hibernate.jdbc.BatchingBatcher.doExecuteBatch(BatchingBatcher.java:70)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:268)\r\n\t... 17 more\r\n{code}\r\n\r\nThe error prevents any data being imported to the data warehouse.\r\n\r\nThis issue is due to a new create event being present in the cloud database which has the same instance uuid but a different event id.\r\n\r\nIt is possible that this issue also occurs with EBS instances that are restarted after the origin creation event has been exported and deleted.","customfield_13200":null,"customfield_11300":null,"customfield_11500":null,"timetracking":{},"customfield_10401":null,"customfield_10005":null,"customfield_10600":null,"customfield_12900":null,"aggregatetimeestimate":null,"attachment":[],"summary":"Data warehouse import can fail after data deleted from cloud","creator":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=sjones","name":"sjones","key":"sjones","accountId":"557058:f5dce1b0-db45-4dd6-a0e8-3cea5effb5ce","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue"},"displayName":"Steve Jones","active":true,"timeZone":"America/Los_Angeles"},"subtasks":[],"reporter":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=sjones","name":"sjones","key":"sjones","accountId":"557058:f5dce1b0-db45-4dd6-a0e8-3cea5effb5ce","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue"},"displayName":"Steve Jones","active":true,"timeZone":"America/Los_Angeles"},"customfield_10000":"3_*:*_1_*:*_440261470_*|*_10001_*:*_2_*:*_109538704_*|*_10002_*:*_1_*:*_40005_*|*_10000_*:*_1_*:*_60959645_*|*_10006_*:*_1_*:*_0_*|*_10003_*:*_2_*:*_3124955389","customfield_12100":"0|i00sxj:","aggregateprogress":{"progress":0,"total":0},"customfield_10001":"2012-11-13T15:00:23.405-0600","customfield_10200":null,"customfield_10002":null,"customfield_10003":["Reporting"],"customfield_10400":null,"customfield_12500":null,"customfield_10116":null,"customfield_11600":["sjones(sjones)","tony(tony)","vasya(vasya)"],"customfield_10117":null,"environment":null,"customfield_11800":null,"customfield_11802":null,"duedate":null,"customfield_11805":null,"progress":{"progress":0,"total":0},"customfield_11806":null,"votes":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/EUCA-3908/votes","votes":0,"hasVoted":false},"comment":{"comments":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/16630/comment/35211","id":"35211","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=tony","name":"tony","key":"tony","accountId":"557058:353956db-b78c-4f3b-9096-f8f58e6709c8","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue"},"displayName":"Tony Beckham","active":true,"timeZone":"America/Tijuana"},"body":"Still reproducible using eucadw:\r\nVersion     : 3.2.0\r\nRelease     : 0.0.80.20121107gitdd0cb22e.el6\r\n\r\n\r\n{code}\r\nroot@eucahost-51-172 ~]# eucadw-import-data -e exportdata2.txt -E -p mypassword\r\n2012-11-13 12:58:22 ERROR | Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-11-13 12:54:42.909000 -08:00:00', '1352840082908', 'PARTI00', 'i-C1CC42BE', 'm1.small', '66FNFMYFB0CLGZIMUWCE2', '01f977e9-5e1b-4144-b574-9fe2586f601e', '4028b32b3af80fdd013afb8d75dd0030') was aborted.  Call getNextException to see the cause.\r\n0 [main] ERROR org.hibernate.util.JDBCExceptionReporter  - Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-11-13 12:54:42.909000 -08:00:00', '1352840082908', 'PARTI00', 'i-C1CC42BE', 'm1.small', '66FNFMYFB0CLGZIMUWCE2', '01f977e9-5e1b-4144-b574-9fe2586f601e', '4028b32b3af80fdd013afb8d75dd0030') was aborted.  Call getNextException to see the cause.\r\n2012-11-13 12:58:22 ERROR | ERROR: duplicate key value violates unique constraint \"reporting_instance_create_events_uuid_key\"\r\n                          |   Detail: Key (uuid)=(01f977e9-5e1b-4144-b574-9fe2586f601e) already exists.\r\n2 [main] ERROR org.hibernate.util.JDBCExceptionReporter  - ERROR: duplicate key value violates unique constraint \"reporting_instance_create_events_uuid_key\"\r\n  Detail: Key (uuid)=(01f977e9-5e1b-4144-b574-9fe2586f601e) already exists.\r\n2012-11-13 12:58:22 ERROR | Could not synchronize database state with session\r\norg.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:262)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:178)\r\n\tat org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)\r\n\tat org.hibernate.event.def.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:51)\r\n\tat org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1206)\r\n\tat org.hibernate.impl.SessionImpl.managedFlush(SessionImpl.java:375)\r\n\tat org.hibernate.transaction.JDBCTransaction.commit(JDBCTransaction.java:137)\r\n\tat org.hibernate.ejb.TransactionImpl.commit(TransactionImpl.java:76)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx$TxState.commit(Entities.java:846)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx.commit(Entities.java:704)\r\n\tat com.eucalyptus.reporting.export.Import.transactional(Import.java:166)\r\n\tat com.eucalyptus.reporting.export.Import.doImport(Import.java:101)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:78)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:70)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.runCommand(ImportCommand.java:45)\r\n\tat com.eucalyptus.reporting.dw.commands.CommandSupport.run(CommandSupport.java:77)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.main(ImportCommand.java:67)\r\nCaused by: java.sql.BatchUpdateException: Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-11-13 12:54:42.909000 -08:00:00', '1352840082908', 'PARTI00', 'i-C1CC42BE', 'm1.small', '66FNFMYFB0CLGZIMUWCE2', '01f977e9-5e1b-4144-b574-9fe2586f601e', '4028b32b3af80fdd013afb8d75dd0030') was aborted.  Call getNextException to see the cause.\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement$BatchResultHandler.handleError(AbstractJdbc2Statement.java:2621)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1837)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:407)\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement.executeBatch(AbstractJdbc2Statement.java:2754)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:616)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.invoke(ProxyStatement.java:100)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.intercept(ProxyStatement.java:57)\r\n\tat org.postgresql.PGStatement$$EnhancerByProxool$$8663693f.executeBatch(<generated>)\r\n\tat org.hibernate.jdbc.BatchingBatcher.doExecuteBatch(BatchingBatcher.java:70)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:268)\r\n\t... 17 more\r\n3 [main] ERROR org.hibernate.event.def.AbstractFlushingEventListener  - Could not synchronize database state with session\r\norg.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:262)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:178)\r\n\tat org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)\r\n\tat org.hibernate.event.def.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:51)\r\n\tat org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1206)\r\n\tat org.hibernate.impl.SessionImpl.managedFlush(SessionImpl.java:375)\r\n\tat org.hibernate.transaction.JDBCTransaction.commit(JDBCTransaction.java:137)\r\n\tat org.hibernate.ejb.TransactionImpl.commit(TransactionImpl.java:76)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx$TxState.commit(Entities.java:846)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx.commit(Entities.java:704)\r\n\tat com.eucalyptus.reporting.export.Import.transactional(Import.java:166)\r\n\tat com.eucalyptus.reporting.export.Import.doImport(Import.java:101)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:78)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:70)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.runCommand(ImportCommand.java:45)\r\n\tat com.eucalyptus.reporting.dw.commands.CommandSupport.run(CommandSupport.java:77)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.main(ImportCommand.java:67)\r\nCaused by: java.sql.BatchUpdateException: Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-11-13 12:54:42.909000 -08:00:00', '1352840082908', 'PARTI00', 'i-C1CC42BE', 'm1.small', '66FNFMYFB0CLGZIMUWCE2', '01f977e9-5e1b-4144-b574-9fe2586f601e', '4028b32b3af80fdd013afb8d75dd0030') was aborted.  Call getNextException to see the cause.\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement$BatchResultHandler.handleError(AbstractJdbc2Statement.java:2621)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1837)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:407)\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement.executeBatch(AbstractJdbc2Statement.java:2754)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:616)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.invoke(ProxyStatement.java:100)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.intercept(ProxyStatement.java:57)\r\n\tat org.postgresql.PGStatement$$EnhancerByProxool$$8663693f.executeBatch(<generated>)\r\n\tat org.hibernate.jdbc.BatchingBatcher.doExecuteBatch(BatchingBatcher.java:70)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:268)\r\n\t... 17 more\r\nError processing command: Could not execute JDBC batch update\r\n2012-11-13 12:58:22 ERROR | Error processing command\r\norg.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update\r\n\tat org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)\r\n\tat org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:262)\r\n\tat org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:178)\r\n\tat org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)\r\n\tat org.hibernate.event.def.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:51)\r\n\tat org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1206)\r\n\tat org.hibernate.impl.SessionImpl.managedFlush(SessionImpl.java:375)\r\n\tat org.hibernate.transaction.JDBCTransaction.commit(JDBCTransaction.java:137)\r\n\tat org.hibernate.ejb.TransactionImpl.commit(TransactionImpl.java:76)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx$TxState.commit(Entities.java:846)\r\n\tat com.eucalyptus.entities.Entities$CascadingTx.commit(Entities.java:704)\r\n\tat com.eucalyptus.reporting.export.Import.transactional(Import.java:166)\r\n\tat com.eucalyptus.reporting.export.Import.doImport(Import.java:101)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:78)\r\n\tat com.eucalyptus.reporting.export.Import.importData(Import.java:70)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.runCommand(ImportCommand.java:45)\r\n\tat com.eucalyptus.reporting.dw.commands.CommandSupport.run(CommandSupport.java:77)\r\n\tat com.eucalyptus.reporting.dw.commands.ImportCommand.main(ImportCommand.java:67)\r\nCaused by: java.sql.BatchUpdateException: Batch entry 0 insert into reporting_instance_create_events (creation_timestamp, timestamp_ms, availability_zone, instance_id, instance_type, user_id, uuid, id) values ('2012-11-13 12:54:42.909000 -08:00:00', '1352840082908', 'PARTI00', 'i-C1CC42BE', 'm1.small', '66FNFMYFB0CLGZIMUWCE2', '01f977e9-5e1b-4144-b574-9fe2586f601e', '4028b32b3af80fdd013afb8d75dd0030') was aborted.  Call getNextException to see the cause.\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement$BatchResultHandler.handleError(AbstractJdbc2Statement.java:2621)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1837)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:407)\r\n\tat org.postgresql.jdbc2.AbstractJdbc2Statement.executeBatch(AbstractJdbc2Statement.java:2754)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:616)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.invoke(ProxyStatement.java:100)\r\n\tat org.logicalcobwebs.proxool.ProxyStatement.intercept(ProxyStatement.java:57)\r\n\tat org.postgresql.PGStatement$$EnhancerByProxool$$8663693f.executeBatch(<generated>)\r\n\tat org.hibernate.jdbc.BatchingBatcher.doExecuteBatch(BatchingBatcher.java:70)\r\n\tat org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:268)\r\n\t... 17 more\r\n{code}","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=tony","name":"tony","key":"tony","accountId":"557058:353956db-b78c-4f3b-9096-f8f58e6709c8","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue"},"displayName":"Tony Beckham","active":true,"timeZone":"America/Tijuana"},"created":"2012-11-13T15:00:23.405-0600","updated":"2012-11-13T15:00:23.405-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/16630/comment/35220","id":"35220","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=sjones","name":"sjones","key":"sjones","accountId":"557058:f5dce1b0-db45-4dd6-a0e8-3cea5effb5ce","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue"},"displayName":"Steve Jones","active":true,"timeZone":"America/Los_Angeles"},"body":"Ensure that this is a cleanly created data warehouse, the test will fail if a DB created prior to the fix is used.","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=sjones","name":"sjones","key":"sjones","accountId":"557058:f5dce1b0-db45-4dd6-a0e8-3cea5effb5ce","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue"},"displayName":"Steve Jones","active":true,"timeZone":"America/Los_Angeles"},"created":"2012-11-13T16:08:44.823-0600","updated":"2012-11-13T16:08:44.823-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/16630/comment/36848","id":"36848","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=vasya","name":"vasya","key":"vasya","accountId":"557058:1c63481a-0634-4ae7-9004-cb0f143e141f","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue"},"displayName":"Vasiliy Kochergin","active":false,"timeZone":"America/Los_Angeles"},"body":"Tested by:\ncreated DW using instruction from https://github.com/eucalyptus/bodega\neureport-export-data -s 1000-01-01 -e 3000-01-01 /tmp/export.xml\neucadw-import-data -e /tmp/export.xml -E -p mypassword\neureport-delete-data -e 3000-01-01\n/etc/init.d/eucalyptus-nc stop\nwait till instance are in the terminated state\n/etc/init.d/eucalyptus-nc start\neureport-export-data -s 1000-01-01 -e 3000-01-01 /tmp/export-2.xml\neucadw-import-data -e /tmp/export-2.xml -E -p mypassword","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=vasya","name":"vasya","key":"vasya","accountId":"557058:1c63481a-0634-4ae7-9004-cb0f143e141f","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/fa832ffa4e20c9aa05ac272214d43196?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dvasya%26avatarId%3D10408%26noRedirect%3Dtrue"},"displayName":"Vasiliy Kochergin","active":false,"timeZone":"America/Los_Angeles"},"created":"2012-12-04T23:59:11.896-0600","updated":"2012-12-04T23:59:11.896-0600"}],"maxResults":3,"total":3,"startAt":0},"customfield_11808":null,"worklog":{"startAt":0,"maxResults":20,"total":0,"worklogs":[]}}}